layouts/shortcodes/blockquote.html

```diff
@@ -0,0 +1,4 @@
+<!-- blockquote -->
+<blockquote class="punchy">
+  <p>{{ trim .Inner " \n" }}</p>
```

Comment:
```suggestion
  <p><center>{{ trim .Inner " \n" }}</center></p>
```

Reply:
I want to do this through CSS.

Also, I want to make sure that if the quote is multiple lines, the _block_ of text is centered, but the text _within_ the block is left aligned.

Gonna take me a bit to work that out because I'm out of my... element... (HTML/CSS joke)... but I like the suggestion. It will look nice.

Reply:
This should be addressed now.